WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -447,6 +471,14 @@ - (void)showPageListForBlog:(Blog *)blog
     [self.navigationController pushViewController:controller animated:YES];
 }
 
+- (void)showPeopleForBlog:(Blog *)blog
+{
+    // TODO: add analytics
```

Comment:
One comment I've been leaving when I see a TODO is to encourage people to attach their name to the TODO as well and the date. E.g. `TODO(kokejb, 10/30/2015):`